titan-compiler/typedecl.lua

```diff
@@ -9,7 +9,7 @@ return function(prefix, types)
             constructors[consname] = function(...)
                 local args = table.pack(...)
                 if args.n ~= #fields then
-                    error("missing arguments for " .. consname)
+                    error("missing arguments for " .. consname .. tostring(args.n) .. tostring(#fields))
```

Comment:
It seems that these were added for debugging but not remover later.
If we want to keep them they should have a space between the numbers at least.